testutil/integration/suites/param_configs.go

```diff
@@ -24,6 +24,7 @@ const (
 	ParamTypeInt64   ParamType = "int64"
 	ParamTypeUint64  ParamType = "uint64"
 	ParamTypeFloat32 ParamType = "float32"
+	ParamTypeDouble  ParamType = "float64"
```

Comment:
For consistency, I prefer `Float64` than `Double`

Reply:
Consistent with what though because go calls it `float64` but protobuf calls it `double`. I was preferring consistency with the protobuf type, do you think we should go the other way with it?